.travis/script.sh

```diff
@@ -90,7 +90,7 @@ build_cc1352() {
     git checkout -- . || die
     git clean -xfd || die
     ./bootstrap || die
-    COMMISSIONER=1 JOINER=1 DHCP6_CLIENT=1 DHCP6_SERVER=1 DNS_CLIENT=1 make -f examples/Makefile-cc1352 || die
+    COMMISSIONER=1 JOINER=1 SLAAC=1 DHCP6_CLIENT=1 DHCP6_SERVER=1 DNS_CLIENT=1 make -f examples/Makefile-cc1352 || die
```

Comment:
It seems you already enabled SLAAC by default.

Reply:
Yes. We ended up with a solution to add it in  `common-switches` and keep it enabled by default for all example makefile (keeping it enabled for all CLI example builds). For internal projects (whic use their own makefiles) this would be disabled by default. 

I still kept `SLAAC=1`  here in travis script to be explicit about it and be safe if in future the default in `common-switches` changes.